packages/cross_file/lib/src/types/html.dart

```diff
@@ -74,8 +77,8 @@ class XFile extends XFileBase {
   // Initializes a Blob from a bunch of `bytes` and an optional `mimeType`.
   Blob _createBlobFromBytes(Uint8List bytes, String? mimeType) {
     return (mimeType == null)
-        ? Blob(<dynamic>[bytes])
-        : Blob(<dynamic>[bytes], mimeType);
+        ? Blob(<JSAny?>[bytes.toJS].toJS)
```

Comment:
Do you need the `<JSAny?>` here and below? You should be able to call `toJS` on a `List<JSUint8Array>`.

Reply:
lints want generics. Added more specific types.